CMakeModules/build_forge.cmake

```diff
@@ -33,12 +34,14 @@ ExternalProject_Add(
     ${byproducts}
     )
 
+ExternalProject_Get_Property(forge-ext binary_dir)
 ExternalProject_Get_Property(forge-ext install_dir)
+SET(forge_bindir_location ${binary_dir}/src/${CMAKE_SHARED_LIBRARY_PREFIX}forge${CMAKE_SHARED_LIBRARY_SUFFIX})
+IF(NOT (${forge_bindir_location} STREQUAL ${forge_bin_location}))
+    MESSAGE(WARNING "Did the forge binary location move? (Have ${forge_bindir_location} vs ${forge_bin_location}")
+ENDIF()
 ADD_LIBRARY(forge SHARED IMPORTED)
-SET_TARGET_PROPERTIES(forge PROPERTIES IMPORTED_LOCATION ${forge_location})
-IF(WIN32)
-    SET_TARGET_PROPERTIES(forge PROPERTIES IMPORTED_IMPLIB ${prefix}/lib/forge.lib)
-ENDIF(WIN32)
```

Comment:
We need the IMPORTED_IMPLIB for windows. You can use the bin directory, but it is needed for windows builds.